packages/r/reproc/xmake.lua

```diff
@@ -0,0 +1,31 @@
+package("reproc")
+
+    set_description("a cross-platform C/C++ library that simplifies starting, stopping and communicating with external programs.")
+    set_homepage("https://github.com/DaanDeMeyer/reproc")
+    set_license("MIT")
+
+    add_urls("https://github.com/DaanDeMeyer/reproc.git")
```

Comment:
please add tar.gz url `https://github.com/DaanDeMeyer/reproc/archive/refs/tags/$(version).tar.gz` and it's sha256 to add_versions

Reply:
Consider it done! :)